### PR TITLE
feat(popover): add popover component

### PR DESCRIPTION
**Pull request by MLKiiwy**
Reviewers: @MLKiiwy @kagnamazian @realdreamer @galagmon @michelvdhoek 

Preview: https://static-staging.usabilla.com/stylabilla/BLUP-139-popover/docs/section-components.html#kssref-components-popover

Specs : https://projects.invisionapp.com/share/9NEFGAXM4#/screens/334094110_Popover_Specs

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+Reviewers: @usabilla/fe-chapter @usabilla/design
+
+Preview: https://static-staging.usabilla.com/stylabilla/{$BRANCH_NAME}/docs/section-components.html#kssref-components-popove
```
Comment (kagnamazian) [inline .github/PULL_REQUEST_TEMPLATE.md]: I don't think you want to put the `components-popove` link in the template

File: src/components/_popover.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
+// Popover
+//
+// Browser support: IE9+ (no transitions), Safari6+, Firefox30+, Chrome30+
+//
+// Markup:
+// <button class="sb-btn sb-btn--primary">Popover anchor (trigger)</button>
+// <div class="sb-modal sb-popover--open" role="dialog" aria-labelledby="popover-content" data-kss-animation-switch="sb-popover--open,sb-popover--close" data-kss-animation-name="Popover open/close">
+//   <div class="sb-popover__backdrop" aria-hidden="true"></div>
```
Comment (kagnamazian) [inline src/components/_popover.scss]: why this div is necessary ?

File: src/components/_popover.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.#{$ns}popover__dialog {
+  position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 0;
+  transition: opacity $sb-timing-s $sb-easing-function;
+
+  .#{$ns}popover--open & {
```
Comment (kagnamazian) [inline src/components/_popover.scss]: what is this rule with `&` ?

File: src/components/_popover.scss (hunk 2 above)
Comment (MLKiiwy) [reply src/components/_popover.scss]: To listen the click outside event and close the popover.

File: src/components/_popover.scss (hunk 3 above)
Comment (MLKiiwy) [reply src/components/_popover.scss]: this rule is active if __dialog is in a child of --open.

**Comment by MLKiiwy** [comment]
@Maruceau please review: https://static-staging.usabilla.com/stylabilla/BLUP-139-popover/docs/section-components.html#kssref-components-popover

**Pull request merged**